go-controller/pkg/network-controller-manager/node_network_controller_manager.go

```diff
@@ -82,26 +81,6 @@ func NewNodeNetworkControllerManager(ovnClient *util.OVNClientset, wf factory.No
 	return ncm, nil
 }
 
-// getOVNIfUpCheckMode check if OVN PortBinding.up can be used
-func (ncm *nodeNetworkControllerManager) getOVNIfUpCheckMode() error {
-	// this support is only used when configure Pod's OVS interface, it is not needed in DPU host mode
-	if config.OvnKubeNode.DisableOVNIfaceIdVer || config.OvnKubeNode.Mode == ovntypes.NodeModeDPUHost {
```

Comment:
For reviewers, this is now pushed down to https://github.com/ovn-org/ovn-kubernetes/pull/3596/files#diff-be824e2fe0b18baca9cdadec443e33f74ecfd0ae1cdcf9a30413b54e3ae5540aR272